### ISSUE TITLE
Bug - Create New Phone Ticket TimeUnits not picking up Data Selected 

### ISSUE THREAD

**Issue by gody01** [znuny/Znuny#464]
### Environment

- OS:           Linux 
- Browser:      Chrome/Firefox/... 
- Znuny version: 

### Expected behaviour
SysConfig  Ticket::Frontend::AccountTime::Dropdown###001-TimeUnits has DataSelected entered.

Input Field shows correct Value (0 in my case)

When New Phone Ticket is created, selected and displayed value should be accepted and ticket created.

### Actual behaviour

When creating New Phone Ticket DataSelected is displayed correctly.

When Submit button is pressed, Label of input field is colored red, and Value has to be reentered. Only for 001-TimeUnits, for 002-TimeUnits it is OK.

On all other tested dialogs it works OK, and picks up default value.

### How to reproduce

Video: https://pannonia.gody.si/uploads/PhoneTicketTimeUnits.webm

1. Go to 'Create New Phone Ticket...'
2. Fill mandatory fields
3. Click on 'Submit'
5. See error at Time Units label

### Additional information

https://pannonia.gody.si/uploads/PhoneTicketTimeUnits.webm

**Comment by gody01** [comment]
In debug mode of Chrome, I can see, that on call Validate_TimeUnits Value="" instead '0' as shown.
It seems, that this error is because it is not pop-up but regular window.

Just checked. Behaviour is same at Create New Email Ticket screen.

**Comment by gody01** [comment]
In Pop-up dialogs thise element has Value assigned:
<input type="hidden" id="TimeUnits" name="TimeUnits" value="5" class="W50pc Validate_TimeUnits Validate_Required " aria-required="true">

In Create Phone/Email Ticket this element has no value assigned:
<input type="hidden" id="TimeUnits" name="TimeUnits" value="" class="W50pc Validate_TimeUnits Validate_Required " aria-required="true">

**Comment by gody01** [comment]
This patch seems to fix the problem. It looks like operator //= does not assign value to Hash reliably.
```
--- Kernel/Output/HTML/Layout/Ticket.pm 2023-06-14 10:18:57.000000000 +0200
+++ Custom/Kernel/Output/HTML/Layout/Ticket.pm  2023-07-25 17:35:29.442550465 +0200
@@ -1209,7 +1209,9 @@
             );
         }

-        $Param{TimeUnits} //= $DefaultTimeUnits;
+#        $Param{TimeUnits} //= $DefaultTimeUnits;
+        $Param{TimeUnits} = $DefaultTimeUnits;
+
         $LocalLayoutObject->Block(
             Name => 'TimeUnits',
             Data => \%Param,

```

**Comment by rkaldung** [comment]
@gody01 Please provide us the Znuny version where this issue can be reproduced.

**Comment by gody01** [comment]
Hello,

Version is 6.5.3 (LTS).
DataSelected (default numbers) has to be entered (I have 0 hours, 5 minutes). Don't use seconds.

If I don't change value in Hours than there is an error.
If I change value in Hours, than no error appears.

Did You see video above, with steps to reproduce ?
In other displays (popup instead of ordinari page for ticket creation) with same DataSelected there is no error, only on two create email ticket and create phone ticket pages.

**Comment by rkaldung** [comment]
internal issue 776

**Comment by jepf** [comment]
Fixed in next 6.5 and 7.0 releases.
